templates/scenarios/sso-tab-js/package.json

```diff
@@ -19,10 +19,10 @@
     },
     "devDependencies": {
         "cross-env": "^7.0.3",
-        "env-cmd": "^10.1.0"
+        "dotenv": "^16.0.3"
     },
     "scripts": {
-        "dev:teamsfx": "env-cmd --silent -f .env.teamsfx.local npm run start",
+        "dev": "react-scripts start",
```

Comment:
Can we just leverage the existing `start` script?

Reply:
Updated the script.